addons/koji/common/src/main/java/org/commonjava/indy/koji/content/KojiContentManagerDecorator.java

```diff
@@ -318,6 +325,27 @@ public Transfer getTransfer( final ArtifactStore store, final String path, final
                         continue;
                     }
 
+                    List<KojiArchiveInfo> jars = new ArrayList<>();
+                    List<KojiArchiveInfo> poms = new ArrayList<>();
+                    List<KojiArchiveInfo> others = new ArrayList<>();
+                    for ( KojiArchiveInfo archive : kojiClient.listArchivesForBuild( build.getId(), session )
+                                                              .getArchives() )
+                    {
+                        if ( "jar".equals( archive.getTypeName().toLowerCase() ) )
```

Comment:
I've just realized we should probably also skip sources.jar and javadoc.jar files, because .they usually don't contain any timestamps so a rebuild would produce different binary jar along with the same sources.jar and that's what we want to detect by this.